src/EnergyPlus/DataAirLoop.hh

```diff
@@ -265,6 +266,7 @@ namespace DataAirLoop {
         Array1D_int ComponentType_Num; // Parameterized (see above) Component Types this
         // module can address
         Array1D_int ComponentIndex; // Which one in list -- updated by routines called from here
+        std::vector <UnitarySystems::UnitarySys *> compPointer;
```

Comment:
@Myoldmopar I have set up these pointers (1 each for zone, air loop, and OA system) to save the factory return. Is there a more generic way to do this so that other models can use this same vector to store their pointer? std::shared, std::unique, etc. See line 269 DataAirLoop.hh, line 397 DataZoneEquipment.hh, line 128 OutdoorAirUnit.hh